src/core/lv_obj.h

```diff
@@ -263,6 +263,13 @@ void lv_obj_clear_state(lv_obj_t * obj, lv_state_t state);
  */
 void lv_obj_set_base_dir(lv_obj_t * obj, lv_bidi_dir_t dir);
 
+#if LV_USE_USER_DATA
+static inline void lv_obj_set_user_data(lv_obj_t * obj, void * user_data)
```

Comment:
Please add some comments to the functions.

Reply:
Added.

Reply:
Thanks!